api/common/catalogue_python_fixtures/fixtures/aws.py

```diff
@@ -0,0 +1,66 @@
+import boto3
```

Comment:
This is duplicated not moved as some of the adapters rely on this as well as the snapshot generator (and it didn't look complicated enough to extract and share).